backend/tracim_backend/lib/core/event.py

```diff
@@ -90,6 +90,7 @@ def _base_query(
         event_id: typing.Optional[int] = None,
         user_id: typing.Optional[int] = None,
         event_types: typing.List[EventTypeDatabaseParameters] = None,
+        exclude_author_ids: typing.List[int] = None,
```

Comment:
This is an `Optional[List...]`:

```suggestion
        exclude_author_ids: typing.Optional[typing.List[int]] = None,
```